libs/map-editor/src/types.ts

```diff
@@ -12,13 +24,86 @@ export enum AreaType {
     Dynamic = "Dynamic",
 }
 
+export enum Direction {
+    Left = "Left",
+    Up = "Up",
+    Down = "Down",
+    Right = "Right",
+}
+
+export const isEntityRawPrefab = z.object({
```

Comment:
At the beginning of working with Zod, we used this weird naming convention to start Zod types with "is".

But most people on the web (and Zod creator itself) is actually naming the Zod const to the same name as the type.

Could you rename this to `EntityRawPrefab` (and all other Zod types too?)